packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -160,6 +160,14 @@ export interface DesignSystem {
     neutralFillToggleActiveDelta: number;
     neutralFillToggleFocusDelta: number;
 
+    /**
+     * The luminance value to base layer recipes on.
+     * Sets the luminance value for the L1 layer recipe in a manner that can adjust to variable contrast.
+     *
+     * Currently defaults to -1 to turn the feature off and use backgroundColor for layer colors instead.
+     */
+    baseLayerLuminance: number; // 0...1

```

Comment:
Correct me if I'm wrong but isn't this also a delta? Therefore should the naming not be `layerLuminanceDelta`?

Reply:
It's not a delta. It's more like a brightness setting from 0-100. Deltas have been offsets on the ramp.